pkg/validator/testutils/validator_tester.go

```diff
@@ -45,6 +45,10 @@ type ValidatorTester struct {
 	ocm validator.OCMClient
 }
 
+type stringKey string
+
+var TestEnvKey = stringKey("TEST_AM0014")
```

Comment:
```suggestion
var TestEnvKey = stringKey("TEST")
```
Let's actually use a generic string like `TEST` because then, even other addons would be able utilise this and have test-specific logic, instead of going through the hassle of creating their own test key and using it the same way.

wdyt :)

Reply:
Yes, makes sense! 👍🏼